parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -71,6 +71,16 @@ $global:CNIConfig = [Io.path]::Combine($global:CNIPath, "config", "`$global:Netw
 $global:HNSModule = [Io.path]::Combine("$global:KubeDir", "hns.psm1")
 
 $global:VolumePluginDir = [Io.path]::Combine("$global:KubeDir", "volumeplugins")
+#azure cni
+$global:NetworkPolicy = "{{WrapAsVariable "networkPolicy"}}"
+$global:VNetCNIPluginsURL = "{{WrapAsVariable "vnetCniWindowsPluginsURL"}}"
+$global:MaxPods = "{{WrapAsVariable "maxPods"}}"
+
+$global:AzureCNIDir = [Io.path]::Combine("$global:KubeDir", "azurecni")
+$global:AzureCNIBinDir = [Io.path]::Combine("$global:AzureCNIDir", "bin")
+$global:AzureCNIConfDir = [Io.path]::Combine("$global:AzureCNIDir", "netconf")
+$global:AzureCNIKubeletOptions = " --network-plugin=cni --cni-bin-dir=$global:AzureCNIBinDir --cni-conf-dir=$global:AzureCNIConfDir"

```

Comment:
This is nice. Can you do me a favor to add a WindowsCNIKubeletOptions for windows cni option to make it more readable?

Reply:
done.